dataflow/src/main/java/org/checkerframework/dataflow/analysis/AbstractAnalysis.java

```diff
@@ -200,6 +200,21 @@ public AnalysisResult<V, S> getResult() {
             assert !n.isLValue() : "Did not expect an lvalue, but got " + n;
             if (!currentNode.getOperands().contains(n)
                     && !currentNode.getTransitiveOperands().contains(n)) {
+                // if n.getTree.getKind is a postfix tree, we check if the target of the assignment
+                // is contained in the operands or not.
+                if (n instanceof AssignmentNode) {
+                    Tree.Kind nKind = n.getTree().getKind();
```

Comment:
the `getKind()` method makes currentNode nullable again (gives `deference.of.nullable` when `currentNode.getOperands()`), and the method is in java library.

Reply:
already make a pr to eisop jdk